### ISSUE TITLE
Deploy only subdirectory, not git root

### ISSUE THREAD

**Issue by plashenkov** [BrunoDeBarros/git-deploy-php#76]
Hi!

Is there a way to deploy only specific subdirectory of git repository, not git root? For example, a complete project to be deployed resides in a directory inside git repository (outside of this dir there are some service files: README, LICENSE, composer.json, .editorconfig, etc.)

For example, in [git-ftp](https://github.com/git-ftp/git-ftp) there is a --syncroot option to specify local root to be deployed.

Is there some sort of similar option for git-deploy-php?

Thank you!

**Comment by BrunoDeBarros** [comment]
No, I'm afraid there isn't such a thing at the moment; that's something that I'd like to add though. Shouldn't be too difficult to add it in, but I haven't had much time lately, I'm afraid.

**Comment by plashenkov** [comment]
I wish I could help you with that but I am not sure I will do that correctly.
Looks like something here should be changed: https://github.com/BrunoDeBarros/git-deploy-php/blob/master/tools/src/Git.php#L46 (git diff subfolder/). Maybe somewhere else?

**Comment by louisluu** [comment]
+1
